doc/content/getting-started/migrating-from-v2/major-changes.md

```diff
@@ -30,7 +30,7 @@ The Packet Broker enables peering between networks, so traffic received by one n
 
 With Packet Broker enabled on both {{% tts %}} and {{% ttnv2 %}}, you can receive traffic on {{% tts %}} without having to re-configure any of your gateways.
 
-> **NOTE**: Packet Broker is enabled on The Things Network Public Community Network and The Things Industries Cloud Hosted.
+> **NOTE**: Packet Broker is enabled on The Things Network Public Community Network and The Things Industries Cloud Hosted. NOT on AWS' marketplace launcher.
```

Comment:
Unnecessary addition; then we can list lots of other deployment models where this is not enabled